app/views/rebels/index.html.slim

```diff
@@ -54,12 +56,14 @@ div(data-controller="datatable" data-datatable-export-title="#{FFaker::Music.son
             td(data-sort="#{rebel.created_at.iso8601}" style="width: 8rem")
               = l(rebel.created_at.to_date)
             td
-              = rebel.status.present? ? rebel.status : "-"
+              = rebel.status.presence || "-"
             td
               - if rebel.willingness_to_be_arrested?
                 .inline-icon
                   = render "layouts/icons/check_square"
-            td(style="width: 10rem")
+            td
+              = rebel.number_of_arrests.presence || "-"
+            td(style="width: 5rem")
```

Comment:
I changed the width here because the icon doesn't require 10rem of space, and the table was starting to look a little squished with the new column.

Reply:
Yes, the table is becoming a real mess. We'll work on that.